### PR TITLE
Scaffold initial v2 doc layout

### PR DESCRIPTION
**Pull request by vdice**
Scaffolds a potential v2 doc layout as proposed in https://github.com/brigadecore/brigade/issues/1483

* I kept the high-level dividers of `intro` and `topics`.  See the corresponding index.md files for both sections on layout (brought over from https://github.com/brigadecore/brigade/issues/1483) and TODOs.  In accordance, I deleted files I figured were not relevant but kept files that are more relevant, though still would need updating.
* I tried to keep actual doc changes to a minimum
* In reviewing, let's look at scaffolding/hierarchy first and foremost... once we're happy with a way forward, we can start creating documentation issues in the backlog that anyone can grab and work on.

Question: Since we are serving the v2 docs in preview, how transparent do we want to be?  Do we keep TODOs/notes inline or do we only add/expose fully-fledged, updated v2 docs (like the updated Quickstart and Install docs from @carolynvs)?  Or somewhere in-between?

TODO

 - [x] Audit files in `docs/static/img` and remove unused assets

### CODE REVIEW

File: docs/content/intro/_index.md [hunk 1]
```diff
@@ -11,16 +11,12 @@ weight: 10
...
 # Getting Started
 
-New to Brigade or to CI systems in general? Well, you came to the right place: read this material to quickly get up and running.
+New to Brigade or looking to quickly get up to speed with v2? Well, you came to the right place: read this material to quickly get up and running.
 
 ## Table of Contents
 
-- [Brigade at a glance](overview)
-- [QuickStart](quickstart)  
+- What's new in v2 (TODO)
```
Comment (vdice) [inline docs/content/intro/_index.md:18]: Added as a possible intro section; not necessarily adamant on it.  Also open to other suggestions for entries here...

File: docs/content/intro/_index.md [hunk 2]
```diff
@@ -11,16 +11,12 @@ weight: 10
...
+- [Brigade at a glance](overview) (TODO: update)
```
Comment (vdice) [inline docs/content/intro/_index.md:19]: Going to work on refreshing this overview doc now...

**Comment by krancour** [comment]
@vdice idk if I'm reading this wrong, but the `config.toml` doesn't appear to be consistent with the outline you posted [here](https://github.com/brigadecore/brigade/issues/1483#issuecomment-879218645).

**Comment by vdice** [comment]
> @vdice idk if I'm reading this wrong, but the config.toml doesn't appear to be consistent with the outline you posted here.

You're right, it doesn't yet match directly, especially the `Topics` section.  I only removed entries (and the corresponding docs) that I knew wouldn't be applicable, but I had yet to add any entries, pending the docs actually be written.  But, in the name of visualizing the hierarchy here, should I set up placeholder entries and doc files in accordance with the vision laid out in https://github.com/brigadecore/brigade/issues/1483#issuecomment-879218645 ?

By the way, first things first, as mentioned in the description above, I thought it made sense to keep the `Intro` and `Topics` sections... the install, overview, quickstart (and maybe a few others) would be under `Intro`, while the architecture, Operator, Admin, Project Developer, Scripting Guide (and others) would go under `Topics`.  More substantial topic sections would get their own index as well, e.g. `/topics/operators/_index.md`... WDYT?

**Comment by krancour** [comment]
> But, in the name of visualizing the hierarchy here, should I set up placeholder entries and doc files in accordance with the vision laid out in #1483 (comment) ?

I would say yes.

> I thought it made sense to keep the Intro and Topics sections... the install, overview, quickstart (and maybe a few others) would be under Intro, while the architecture, Operator, Admin, Project Developer, Scripting Guide (and others) would go under Topics

Make sense to me. 👍

File: docs/content/intro/_index.md (hunk 1 above)
Comment (vdice) [reply docs/content/intro/_index.md]: (Removed; we can always add at a later date.)

File: docs/content/intro/_index.md (hunk 2 above)
Comment (vdice) [reply docs/content/intro/_index.md]: See https://github.com/brigadecore/brigade/pull/1493

**Comment by vdice** [comment]
Looking for help around the bottom nav links.  They seem to be auto-configured to follow alphabetical order.  I was under the impression that the `weight`s specified in the [config.toml](https://github.com/brigadecore/brigade/blob/21e67dbe7fdaec44f62cb86c360fe519e5feb87f/docs/config.toml) would dictate ordering, but it doesn't seem to be.

In the screenshot below, taken of the Topics index, I was hoping for the next link to be to the Design doc (`/topics/design.md`) as opposed to the Administrators section.

Screen Shot 2021-07-15 at 9 42 37 AM

**Review by krancour — APPROVED** [review]

**Pull request merged**